src/lib/ModalWindow.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+import { palette, style } from '../constants';
+
+const WindowDiv = styled.div`
+  position: relative;
+  width: ${(p) => p.width};
+  height: ${(p) => p.height};
+  border-radius: 5px;
+	background-color: ${style.BACKGROUND.main};
+	box-shadow: ${style.SHADOW.big};
+	padding: 0;
+`;
+
+const WindowCloseButton = styled.span`
```

Comment:
Could you make this a button? Otherwise it's not accessible (e.g. not focusable).

Reply:
You're right. Fixed